src/Chapter2.hs

```diff
@@ -336,7 +337,9 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList a b ls = if b - a > 0
```

Comment:
This is an interesting check!
However, it won't be correct if `a` is negative. Perhaps, it makes sense to check `a` and `b` on negativity 🙂

Reply:
If i get negative numbers for `a` and `b`, what kind of list should I return? A reversed sub-list?

Reply:
An empty list is okay in that case 🙂